### DISCUSSION TITLE
`Response.links` not documented

### DISCUSSION THREAD

**Discussion by lsloan** [encode/httpx#2770] [General]
I know httpx is mostly compatible with "requests" and that `Response.links` exists in both.  However, it's not explained in httpx's documentation.  I'm teaching some people to use httpx for making API requests and when I told them to use `Response.links` they were mystified because it's not documented.

Should I open an issue to ask for documentation updates?

(I'm asking in a discussion because I was about to open an issue, but the issue template recommended using a discussion first.)

**Comment by lovelydinosaur** [comment]
Would you like to make a documentation suggestion for this?
